server/api/crud/model_monitoring/model_endpoints.py

```diff
@@ -504,11 +504,10 @@ def list_model_endpoints(
                 # Add the `ModelEndpoint` object into the model endpoints list
                 endpoint_list.endpoints.append(endpoint_obj)
         except mlrun.errors.MLRunInvalidMMStoreType as e:
-            #
             logger.debug(
                 "Failed to list model endpoints because store connection is not defined."
-                " Returning an empty list of model endpoints.\n"
-                f"Error: {mlrun.errors.err_to_str(e)}"
+                " Returning an empty list of model endpoints.",
+                error=mlrun.errors.err_to_str(e),
             )
```

Comment:
DRY :) 
seems like a potential class helper function to raise such issue isn't it?